src/main/java/net/projectzombie/region_rotation/modules/RegionState.java

```diff
@@ -36,16 +39,16 @@
     private String rotateBroadcastMessage = null;
 
     public RegionState(final String regionName,
```

Comment:
Could you write a function that gets all the Chests in the region into an ArrayList<Chest>? That would be super helpful for my CarePackageDrops plugin